gel/_internal/_save.py

```diff
@@ -1625,6 +1675,18 @@ def _traverse(obj: GelModel) -> None:
                         # see the above comment.
                         multi_prop_commit_recursive(linked)
 
+            if new_obj_id := self.new_object_ids.get(id(obj)):
```

Comment:
yeah, rebase is needed. Basically with the latest `master` you'd just `self.new_object_ids.get(obj)`